src/test/java/com/redhat/labs/lodestar/service/ProjectStructureServiceTest.java

```diff
@@ -0,0 +1,160 @@
+package com.redhat.labs.lodestar.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.BDDMockito.given;
+
+import java.util.Arrays;
+
+import javax.inject.Inject;
+import javax.ws.rs.core.Response;
+
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import com.redhat.labs.lodestar.models.Engagement;
+import com.redhat.labs.lodestar.models.gitlab.Group;
+import com.redhat.labs.lodestar.models.gitlab.Namespace;
+import com.redhat.labs.lodestar.models.gitlab.Project;
+import com.redhat.labs.lodestar.rest.client.GitLabService;
+import com.redhat.labs.lodestar.utils.GitLabPathUtils;
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.junit.mockito.InjectMock;
+
+@QuarkusTest
+public class ProjectStructureServiceTest {
+
+    @Inject
+    ProjectStructureService psService;
+
+    @InjectMock
+    @RestClient
+    GitLabService gitLabService;
+
+    // New Customers and Projects
+
+//    @Test
```

Comment:
is this supposed to be left in?

Reply:
yeah, this one should be removed.  i will try to do that today if i can.